internal/controllers/provisioningrequest_hwprovision_test.go

```diff
@@ -800,7 +801,7 @@ func verifyNodeStatus(ctx context.Context, c client.Client, nodes []*hwv1alpha1.
 		updatedNode := &hwv1alpha1.Node{}
 		Expect(c.Get(ctx, client.ObjectKey{Name: node.Name, Namespace: node.Namespace}, updatedNode)).To(Succeed())
 		switch updatedNode.Spec.GroupName {
-		case "master":
+		case "controller":
```

Comment:
We should have these strings as constants in the test file with an appropriate name, rather than literals pasted everywhere. It would make it clearer that "worker" in one spot is the nodegroup name, and the role in another.